api/src/main/java/org/openmrs/module/sync/SyncUtil.java

```diff
@@ -248,7 +250,7 @@ public static void setProperty(Object o, String propName, Object propVal) throws
 		Object[] setterParams = new Object[] { propVal };
 		
 		log.debug("getting setter method");

```

Comment:
I needed to change the signature of this method, since with null values there is no value class.  And it didn't seem necessary for the functionality to work.

Reply:
I didn't check exactly what the Apache Commons PropetyUtils.getWriteMethod, fwiw, but I'll assume you checked and it's comparable to the old method.

Reply:
I probably could have checked more, but the unit tests all pass